dockerfile

```diff
@@ -16,6 +16,7 @@ LABEL maintainer="https://teamdigitale.governo.it"
 
 WORKDIR /usr/src/app
 COPY /package.json /usr/src/app/package.json
+COPY /.env /usr/src/app/.env
```

Comment:
We don't need this, the docker image will not contain the env vars, they will be passed to the environment by kubernetes